test/StatObjectTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Storj\Uplink\Test;
+
+use DateTimeImmutable;
+use PHPUnit\Framework\TestCase;
+use Storj\Uplink\Exception\Object\ObjectNotFound;
+
+class StatObjectTest extends TestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        $project = Util::emptyAccess()->openProject();
+        $project->createBucket('phpunit');
+
+        $upload = $project->uploadObject('phpunit', 'StatPrefix/StatObject');
+        $upload->setCustomMetadata(['myhouse' => 'myrules']);
+        $upload->write(random_bytes(55));
+        $upload->commit();
+    }
+
+    public function testStatObject(): void
+    {
+        $objectInfo = Util::project()->statObject('phpunit', 'StatPrefix/StatObject');
+
+        self::assertEquals('StatPrefix/StatObject', $objectInfo->getKey());
+        self::assertEquals(['myhouse' => 'myrules'], $objectInfo->getCustomMetadata());
+        self::assertFalse($objectInfo->isPrefix());
+
+        $systemMetaData = $objectInfo->getSystemMetadata();
+
+        self::assertEquals(55, $systemMetaData->getContentLength());
+        self::assertEquals(null, $systemMetaData->getExpires());
+        self::assertLessThan(
+            60,
+            abs(time() - $systemMetaData->getCreated()->format('U'))
+        );
+    }
+
+    // appearantly this isn't implemented
+//    public function testStatPrefix(): void
+//    {
+//        $objectInfo = Util::project()->statObject('phpunit', 'StatPrefix');
```

Comment:
I'm wondering what does it return?
does `StatPrefix/` produce the same result?

Reply:
Good feedback. 

It returns an error that the object does not exist. Both with `/` and without.

I've uncomment the test, duplicated it for both cases, and added an assertion for the exception.